digitalforms-api/src/test/java/ca/bc/gov/open/pssg/rsbc/digitalforms/model/StatusInfoResponseTests.java

```diff
@@ -38,6 +41,21 @@ public void testObj() {
 		Assertions.assertEquals("date", infoResponse.getIRPInfo().getEffectiveDate());
 		Assertions.assertEquals("iRP", infoResponse.getIRPInfo().getIRPStatus());
 		Assertions.assertEquals("name", infoResponse.getIRPInfo().getSurnameNm());
+		
+		ProhibitionStatus status = new ProhibitionStatus();
```

Comment:
I'll be committing some code today which will conflict with this class (and possibly some others). We'll deal with it when it comes. The prohibition status has changed.